scripts/metadata_comparison/metadata_comparison/lib/papi/papi_clients.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+#
+# Lazy initializer for various PAPI clients so that we only create them when we need them
+#
+
+from googleapiclient.discovery import build as google_client_build
+from metadata_comparison.lib.operation_ids import *
+import logging as log
```

Comment:
`logging` is clearly a module. `log` can also be a log. This alias makes the code more confusing. And it only saves 4 characters...
(Yes, this is an opinionated statement.)

Reply:
Yeah, it looked weird to me to. I copy/pasted this from the internet so I assumed it was something python-y, but I'm happy to take your comment as an excuse to do the more logical looking thing!